client/asset/btc/btc.go

```diff
@@ -970,9 +974,10 @@ func (btc *ExchangeWallet) Swap(swaps *asset.Swaps) ([]asset.Receipt, asset.Coin
 	}
 
 	// Delete the UTXOs from the cache.
-	for _, pt := range pts {
-		delete(btc.fundingCoins, pt)
+	for _, spentOp := range spentOps {
+		delete(btc.fundingCoins, spentOp.pt)
 	}
+	btc.wallet.LockUnspent(true, spentOps)
```

Comment:
I believe bitoind unlocks these on spend, but I don't really object to trying it anyway.  https://github.com/bitcoin/bitcoin/pull/13160  @buck54321 Is that what you found in testing this?  @itswisdomagain if this is right and we are certain outpoints are being unlocke automatically and immediately, just replace with a comment please.

Reply:
Yes. **bitcoind** unlocks on spend. This would likely generate an error.

Reply:
Simnet trading tests no longer have balance check issues, so seems both wallets are unlocking spent coins as expected. Will replace with a comment.

Reply:
Do we know if dcr also auto-unlock coins on spend? Last, I checked, it did but I'm getting weary of trusting these asset backend behaviours when it comes to coin unlocking. We currently force coin unlocking after sending swaps for dcr, so wondering if I should as well replace that with a comment.
https://github.com/decred/dcrdex/blob/ceb666181576a60572d09bc7ff42f1c066fb598f/client/asset/dcr/dcr.go#L1027-L1031

Reply:
dcrwallet does not. We must do it manually.